contracts/oracles/EtherFi.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+import { OracleInterface } from "../interfaces/OracleInterface.sol";
+import { IWeETH } from "../interfaces/IWeETH.sol";
+import { IEETH } from "../interfaces/IEETH.sol";
+import { ensureNonzeroAddress } from "@venusprotocol/solidity-utilities/contracts/validators.sol";
+import { EXP_SCALE } from "@venusprotocol/solidity-utilities/contracts/constants.sol";
+
+/**
+ * @title EtherFiOracle
+ * @author Venus
+ * @notice This oracle fetches the price of eETH and weETH assets
+ */
+contract EtherFiOracle is OracleInterface {
+    /// @notice Address of weETH
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    IWeETH public immutable weETH;
+
+    /// @notice Address of eETH
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    IEETH public immutable eETH;
+
+    /// @notice Address of Resilient Oracle
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    OracleInterface public immutable RESILIENT_ORACLE;
+
+    /// @notice Set this as asset address for native token on each chain.
+    address public constant NATIVE_TOKEN_ADDR = 0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB;
+
+    /// @notice Constructor for the implementation contract.
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor(address _weETH, address _eETH, address resilientOracleAddress) {
+        ensureNonzeroAddress(_weETH);
+        ensureNonzeroAddress(_eETH);
+        ensureNonzeroAddress(resilientOracleAddress);
+        weETH = IWeETH(_weETH);
+        eETH = IEETH(_eETH);
+        RESILIENT_ORACLE = OracleInterface(resilientOracleAddress);
+    }
```

Comment:
If we are using underscore in the param name then we should use it in all the params.
```suggestion
    constructor(address _weETH, address _eETH, address _resilientOracleAddress) {
        ensureNonzeroAddress(_weETH);
        ensureNonzeroAddress(_eETH);
        ensureNonzeroAddress(_resilientOracleAddress);
        weETH = IWeETH(_weETH);
        eETH = IEETH(_eETH);
        RESILIENT_ORACLE = OracleInterface(_resilientOracleAddress);
    }
```